media-gfx/kseexpr/patches/kseexpr-4.0.4.0.patchset

```diff
@@ -0,0 +1,46 @@
+From 6ba0cb9d62c75e6db7ca5d8aa63912ff08a19411 Mon Sep 17 00:00:00 2001
+From: Luc Schrijvers <begasus@gmail.com>
+Date: Wed, 26 Mar 2025 17:29:18 +0100
+Subject: We don't do -ldl
+
+
+diff --git a/src/KSeExpr/CMakeLists.txt b/src/KSeExpr/CMakeLists.txt
+index 27549b8..2727f80 100644
+--- a/src/KSeExpr/CMakeLists.txt
++++ b/src/KSeExpr/CMakeLists.txt
+@@ -82,7 +82,9 @@ if (NOT WIN32)
+     if (NOT APPLE)
+         set_source_files_properties(interpreter.cpp PROPERTIES COMPILE_OPTIONS "-rdynamic")
+     endif()
++    if (NOT HAIKU)
+     target_link_libraries(KSeExpr "dl")
```

Comment:
If they had used `${CMAKE_DL_LIBS}` instead of the hardcoded `"dl"`, this wouldn't have to be patched (if you are planning to upstream this).

Reply:
Upstreamed, thanks! :)